docs/pages/identity-governance/access-requests/resource-requests.mdx

```diff
@@ -261,450 +252,15 @@ Getting updated certificates...
 iot:~ alice$
 ```
 
-### Restrict the resources a user can request access to
```

Comment:
I can understand moving purely tsh related content to the clients section but it seems that this removes a big chunk of the documentation that explains how to configure access to various resources which is useful. Is this content available also elsewhere or is it just gone?

Reply:
This section has ended up becoming a series of examples of Teleport roles with no relationship to Access Requests, making the page long and convoluted. We could preserve the section introduction (to add detail about `search_as_roles`) while removing the unnecessary H4 sections, linking to different access controls pages we already have in the docs.

Reply:
On second thought, this section assumes that the guide shows the reader how to use `search_as_roles` to limit the resources a user can request access to, but there's actually no mention of `search_as_roles` before this point. The "Next steps" section includes a link to the Access Request configuration reference, which does explain `search_as_roles`. It might make sense to keep all `search_as_roles` content in the configuration guide as it is now.